R/helpers_ggbetween_t_subtitles.R

```diff
@@ -142,9 +141,9 @@ subtitle_t_parametric <- function(data,
     parameter = stats_df$parameter[[1]],
     p.value = stats_df$p.value[[1]],
     effsize.text = effsize.text,
-    effsize.estimate = effsize_df[[3]][[1]],
-    effsize.LL = effsize_df$conf.int[[1]],
-    effsize.UL = effsize_df$conf.int[[2]],
+    effsize.estimate = effsize_df$Estimate,
```

Comment:
With the new changes, shouldn't this (L144-146)
```r
effsize.estimate = effsize_df$Estimate,
effsize.LL = effsize_df$LowerCI,
effsize.UL = effsize_df$UpperCI,
```
instead be
```r
effsize.estimate = effsize_df$estimate,
effsize.LL = effsize_df$conf.low,
effsize.UL = effsize_df$conf.high,
```

Reply:
Yes changed